db/migrate/20191016190325_change_rating_body_null_default.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeRatingBodyNullDefault < ActiveRecord::Migration[5.2]
+  def up
+    change_column_null :reviews, :body, false
+    change_column_default :reviews, :body, ""
```

Comment:
Do you really want your body to be set to "" by default? This would allow a review with a blank body into the database (if it somehow got past your model level validation)

Reply:
Yes, that is correct.

Reply:
Unfortunately that is not correct :) 
If you have an optional column (per the requirements), you do not want to set a default. The difference between storing an empty string and a null value at scale isn't insignificant. 

Please update